draft-ietf-mls-protocol.md

```diff
@@ -1678,19 +1675,19 @@ ciphersuite in use.
 
 # Group Creation
 
-A group is always created with a single member, the "creator".  The other
-members are added when the creator effectively sends itself an Add proposal and
-commits it, then sends the corresponding Welcome message to the new
-participants.  These processes are described in detail in {{add}}, {{commit}},
+A group is always first created with a single member, the "creator".  Other
+members are added when the creator effectively sends itself an _Add_ proposal
+and commits it, and then sends the corresponding _Welcome_ message to the
```

Comment:
Two options:
- sends itself an Add and commits, then (how it was)
- sends itself an Add, commits, and then